modules/ding_tabroll/js/jquery-ui-tabs-rotate.js

```diff
@@ -1,4 +1,6 @@
 ;(function($){
+	'use strict';
```

Comment:
[This file is third party code](https://github.com/cmcculloh/jQuery-UI-Tabs-Rotate) and should not be modified. If anything it should be removed from the repository, downloaded using Drush Make and loaded through libraries API.

Reply:
Yes, I see. Reverted the library. It's outside the scope to move it to libraries right now. :-)